package.json

```diff
@@ -42,6 +42,7 @@
     "color-convert": "^2.0.1",
     "crc-32": "^1.2.0",
     "currency-codes": "^2.1.0",
+    "cypress-repeat": "^2.3.3",
```

Comment:
**suggestion**: move this dependency to optional

We should keep `cypress-repeat` and other cypress/e2e deps inside `optionalDependencies` in pkg.json. 

Reference: https://docs.npmjs.com/cli/v9/configuring-npm/package-json#optionaldependencies

Reply:
ok